playbooks/acs.yml

```diff
@@ -268,6 +271,7 @@
       when: acs.edition == "Enterprise"
   post_tasks:
     - name: Update installation status file with Trouter
+      when: acs.edition == "Enterprise"
```

Comment:
were they leaving an empty file with community or something?

Reply:
No actually it was setting the trouter version set in `group_vars/all.yml` as it is a root element and the community extra vars file is not nulling it.